string zone
roah_rsbb/Score score
---
